plugin/tests/test-extractor.php

```
@@ -0,0 +1,369 @@
+        $this->test(
+            "Different IDs for same key in different subfolders (standalone-apps)",
+            strpos($output, 'id="unraid-standalone-apps-app-entry"') !== false
+        );
+        $this->test(
+            "Different IDs for same key in different subfolders (other)",
+            strpos($output, 'id="unraid-other-app-entry"') !== false
+        );
+        $appEntryCount = substr_count($output, 'id="unraid-') - substr_count($output, 'id="unraid-ui-');
+        $this->test(
+            "Both app-entry scripts are present with unique IDs",
+            preg_match_all('/id="unraid-[^"]*app-entry"/', $output, $matches) === 2
```

PHPMD (2.15.0)

Avoid unused local variables such as '$matches'. (Unused Code Rules)

(UnusedLocalVariable)